packages/deployer/test/fixture-projects/sample-project/test/AnotherModule.test.js

```diff
@@ -1,28 +1,25 @@
 const hre = require('hardhat');
 const assert = require('assert');
 const { ethers } = hre;
-const { getProxyAddress } = require('../../../../utils/deployments');
+const { getProxyAddress } = require('@synthetixio/deployer/utils/deployments');
 const { printGasUsed } = require('@synthetixio/core-js/utils/tests');
-const { bootstrap, initializeSystem } = require('./helpers/initializer');
 const { findEvent } = require('@synthetixio/core-js/utils/events');
+const bootstrap = require('./helpers/bootstrap');
 
 describe('AnotherModule', () => {
-  bootstrap();
+  const { info, init } = bootstrap();
```

Comment:
Now  we are using a custom instance name (`test`), so we have to give it to `deployments` util